monthly-exams/exam-1/src/App.jsx

```diff
@@ -0,0 +1,23 @@
+import React, { useState } from 'react';
+
+import './App.css';
+
+const colors = ['red', 'green', 'blue', 'purple', 'orange', 'black'];
+
+function App() {
+  const [colorIndex, setColor] = useState('');
+  const buttonColor = colors[colorIndex];
+  const onButtonClick = () => {
+    //wrapper(buttonColor)
+  };
+
+  return (
+    <div className={`wrapper ${buttonColor}`}>
+      <div>
+        <button className="button" onClick={onButtonClick}></button>
```

Comment:
You are nearly there, what you are missing is `Array.map`, check my answer is see how I do that